### PR TITLE
Connection to server (REST)

### PR DESCRIPTION
**Pull request by domanttas**
Added Rest Service class;
Added ImageObject model;
Added method to return list of ImageObject from server's DB table 'DatabaseImageElements'

Note: it contains URI of image in Azure Cloud, so we can work with that as a photo.

### CODE REVIEW

File: WhoIsThat/WhoIsThat/Connections/RestService.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            Client = new HttpClient();
+        }
+
+        /// <summary>
+        /// Calls ImageObjectElementController in backend and returns list of image objects
+        /// </summary>
+        /// <returns>List of ImageObject instances</returns>
+        public async Task<List<ImageObject>> GetImageObjects()
+        {
+            string restUrl = "https://whoisthatserverstorage2.azurewebsites.net/api/images/all";
```
Comment (tomasmuzas) [inline WhoIsThat/WhoIsThat/Connections/RestService.cs]: You should store endpoints in a config file of sorts. You could possibly pass it as a parameter or at least set is as `RestService` base

File: WhoIsThat/WhoIsThat/Connections/RestService.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            var uri = new Uri(string.Format(restUrl, string.Empty));
+
+            var response = await Client.GetAsync(uri);
+            if (response.IsSuccessStatusCode)
+            {
+                var content = await response.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<List<ImageObject>>(content);
```
Comment (tomasmuzas) [inline WhoIsThat/WhoIsThat/Connections/RestService.cs]: That's actually a pretty solid and clean way to make a HTTP request! 👍

File: WhoIsThat/WhoIsThat/Connections/RestService.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            }
+
+            else
+            {
```
Comment (tomasmuzas) [inline WhoIsThat/WhoIsThat/Connections/RestService.cs]: Except this part 😃

Comment (domanttas) [reply WhoIsThat/WhoIsThat/Connections/RestService.cs]: I really appreciate the reviews, thanks!

**Pull request merged**